Sources/StreamVideoSwiftUI/CallView/VideoParticipantsView.swift

```diff
@@ -58,33 +58,53 @@ public struct VideoParticipantsView<Factory: ViewFactory>: View {
             }
         }
         .onRotate { newOrientation in
-            orientation = UIApplication.shared.windows.first?.windowScene?.interfaceOrientation ?? .unknown
+            switch newOrientation {
```

Comment:
It seems like the `UIApplication.shared.windows.first?.windowScene` doesn't return the expected orientation when requested here. Instead we translate the deviceOrientation to interfaceOrientation.